frontend/src/lib/components/Support/SupportModal.tsx

```diff
@@ -40,6 +40,31 @@ const SUPPORT_TICKET_KIND_TO_PROMPT: Record<SupportTicketKind, string> = {
     feedback: 'What feedback do you have?',
     support: 'What can we help you with?',
 }
+const SUPPORT_TICKET_KIND_TO_PROMPT_EXTENDED_1: Record<SupportTicketKind, string> = {
+    bug: "What's the bug?",
+    feedback: 'Is your feedback related to a problem? (make sure to include links if relevant)',
+    support: 'What can we help you with? (make sure to include links if relevant)',
+}
+const SUPPORT_TICKET_KIND_TO_PROMPT_EXTENDED_2: Record<SupportTicketKind, string> = {
+    bug: 'How to reproduce?',
+    feedback: "Describe the solution you'd like?",
+    support: 'What is happening?',
+}
+const SUPPORT_TICKET_KIND_TO_PROMPT_EXTENDED_3: Record<SupportTicketKind, string> = {
+    bug: 'Additional context? (make sure to include links if relevant)',
+    feedback: "Describe alternatives you've considered.",
+    support: 'What do you expect to happen?',
+}
```

Comment:
This is hard to read/extend imho.
You'd more likely want to create a type for this sth like (think about this a bit first, not sure if I provided the best approach
```
SupportTicketFormKind:
  inputFields: Field[]
  ...
```

then have:
```
const BUG_TICKET: SupportTicketFormKind(inputFields = [
  Field(prompt = "Description of the Bug", placeholder = "make sure to include links and attachments if relevant", ticketKey = 'description'),
  Field(prompt = "How to reproduce?", ...)
]
```
^ that way we could easily add / remove fields from the form in the future. 
I'd encourage you to keep it the same for all the types not depending on the area, if needed we can create a new type called 'MinimalTicket' or whatever.

Reply:
i like this idea